.github/workflows/request-manager-e2e-test.yml

```diff
@@ -0,0 +1,19 @@
+name: "@trezor/request-manager"
+
+on:
+  pull_request:
+    paths:
+      - "packages/request-manager/**"
```

Comment:
would be great if there was an option to have this "paths" automated. something that it would build dependency tree and decide. something similar to nx

Reply:
Yes, that would be great. Probably that could be achieve with a script that would check if paths of all the dependencies from the monorepo are included here and that script would be call as a test. :man_shrugging:

Reply:
If it was possible to pass a script that would evaluate here instead of array of strings that would be great, but I am afraid this is not possible